_posts/2019-04-14-ferryman-post.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title:  "Our First Game Jam!"
+date:   2019-04-14 14:55:00 -0500
+categories: blog
+author: Aaron
+author_twitter: shamshirz
+image: /assets/ferryman-glass.png
+---
+_48 hour sprint to Unity greatness_
+[![Ferryman image]({{ "/assets/ferryman-glass.png" | relative_url }}){:class="excerpt-image"}]({{ site.baseurl }}{% post_url 2019-04-10-ferryman %})
+<!-- Ends the excerpt text, it includes the image -->
+
+
+How it works.
+
+* Start Friday.
+* Build a game that is loosely based on the theme of "Spirits".
+* Stop working on Sunday.
+
+This was the first time we've joined a challenge like this, so we decided to increase the difficulty by making it our first published game made in Unity. We broke up into 3 roles for the most part; Aaron on art; Sam building the C# game logic; RJ tying everything together in Unity. We each learned a lot about our roles, and how different they are from some of the other games we made.
+
+
+## Unity
+
+The built in collaboration tool was surprisingly great! We are accustomed [Git](https://github.com/SylverStudios) at work and for personal projects, but we tried the Unity built in collaboration tool and ended up sticking with it. There were a couple bumps along the way, but we highly recommend trying it out.
+
+We made a heavily menu based game that would have been easy for us in React, or Elm, but in Unity we were blind. After a good amount of fighting with C#, we did end up with a couple working pages, but I expect there is a much easier way to build UI. C# isn't to blame for that, but we unanimously enjoy the languages that we write usually, but the support of Unity makes C# worth it.
+
+
+## Design
+
+Art takes so much longer than you expect it to! We worked on the project for about 15 hours, and it's crazy how few assets came out of that. In retrospect, there is also so much more to the aesthetic of your game than just the few images of game pieces. The designer's role can encompass the entire feel of the game. With [The Ferryman]({{ site.baseurl }}{% post_url 2019-04-10-ferryman %}) we had a lot of text, menus, and backgrounds.
```

Comment:
May be able to link the first and second sentences better, like instead of "In retrospect", perhaps: "The effort was worth it though; we came away with a new appreciation for how much of the value and experience of a game come from just a few sprites of game pieces".

Reply:
What are you trying to say with the final two sentences? That here design was _less_ important because of how much we relied on test, menus and backgrounds? That the designer's role can, and _should_ encompass the entire feel of the game, but couldn't here because we made a simulation game?

Reply:
Good point, I was trying to say the opposite! I was trying to say that the text, menus, and backgrounds are essential to good design and because it was such a big part of our game, it was very obvious how lacking in design the finished product was.